verticapy/performance/vertica/collection_metadata.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
should it be in another folder named "collection" and we can then rename this file metadata.py?